lib/AST/NameLookup.cpp

```diff
@@ -1067,6 +1068,13 @@ void MemberLookupTable::addMember(Decl *member) {
   // Add this declaration to the lookup set under its compound name and simple
   // name.
   vd->getFullName().addToLookupTable(Lookup, vd);
+
+  // Add this declaration to the lookup set under any @_implements name as well
```

Comment:
How do you prevent name lookup on concrete types from finding the decl? We only want to see the implements name while doing conformance checks no?